app/models/room_request.py

```diff
@@ -92,10 +101,24 @@ def __repr__(self):
                 f'Full Bathroom: {self.full_bathroom}\n'
                 f'Pack n Play: {self.pack_n_play}\n>')
 
-    def __str__(self):
-        return self.__repr__()
+        def generate_fake(count=5, **kwargs):
+            """Generate fake room requests for testing."""
+            from sqlalchemy.exc import IntegrityError
+            from random import seed, choice
+            from faker import Faker
 
+            fake = Faker()
 
-#class Guest(db.Model):
-#    # TODO
-#    pass
\ No newline at end of file
+            seed()
+            for i in range(count):
+                rr = RoomRequest(
+                        first_name = fake.first_name(),
+                        last_name = fake.last_name(),
+                        email = fake.email(),
+                        patient_full_name = fake.full_name(),
+                        **kwargs)
```

Comment:
I think we should add fake data for all the columns! That way we can more easily test front-end and we don't have to manually enter this data everytime through `**kwargs`.